core/src/components/progress-bar/progress-bar.tsx

```diff
@@ -57,10 +57,35 @@ export class ProgressBar implements ComponentInterface {
    */
   @Prop({ reflect: true }) color?: Color;
 
+  /**
+   * Set to `"round"` for a tab-button with rounded corners, or `"rectangular"`
+   * for a tab-button without rounded corners.
```

Comment:
Please update the comment appropriately when copying and pasting.

```suggestion
   * Set to `"round"` for a progress bar with rounded corners, or `"rectangular"`
   * for a progress bar without rounded corners.
```